libs/video.liq

```diff
@@ -27,6 +27,14 @@ end
 def video.add_image(~id="", ~width=int_getter(-1), ~height=int_getter(-1), ~x=int_getter(0), ~y=int_getter(0), ~file, s)
   image =
     if getter.int.is_constant(x) and getter.int.is_constant(y) and getter.int.is_constant(width) and getter.int.is_constant(height) then
+      width = to_int_getter(width)
```

Comment:
Isn't `to_int_getter` working even if `getter.int.is_constant(width)` is constant?

Reply:
Actually I'm not sure what this code means..